src/components/core/Button/Button.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import classes from './Button.module.scss';
+
+interface ButtonProps {
+  content: string;
+  action(): void;
+}
+const Button: React.FC<ButtonProps> = (props) => {
```

Comment:
Need to destructure props based on ButtonProps
({ content, action }: ButtonProps)